### PR TITLE
[Issue #950] remove tablet view

### PR DESCRIPTION
**Pull request by ejnshtein**
JS isMobile.tablet API has been removed.

Changes in styles are made to fix some issues on tablets screens <1024px size.

Tested in Chrome Devtools iPads emulation.
Screenshot 2020-09-24 at 17 56 22
Screenshot 2020-09-24 at 17 58 47
Screenshot 2020-09-24 at 18 00 02

### CODE REVIEW

File: src/app/style/abstract/_media.scss [hunk 1]
```diff
@@ -10,25 +10,46 @@
...
 @mixin before-desktop {
-    @media (max-width: 1024px) {
+    @include mobile {
         @content;
     }
 }
 
 @mixin tablet {
-    @media (max-width: 1024px) and (min-width: 768px) {
+    @media (min-width: 768px)
```
Comment (yeegor) [inline src/app/style/abstract/_media.scss:27]: Why has this change been made?

**Review by yeegor — CHANGES_REQUESTED** [review]
Unable to start the application with these changes.
image

**Comment by ejnshtein** [comment]
> Unable to start the application with these changes.
> image

Don't forget to symlink theme in vendor.

File: src/app/style/abstract/_media.scss (hunk 1 above)
Comment (ejnshtein) [reply src/app/style/abstract/_media.scss:27]: To avoid media query overlap max-width should 1 px smaller than target size.

**Comment by yeegor** [comment]
@ejnshtein it should work without symlinking anything from vendor in core mode. Please fix this.

**Review by yeegor — CHANGES_REQUESTED** [review]
Left my comment above.

**Comment by ejnshtein** [comment]
It won't work without symlink because it's linking _media.scss from the vendor.
If you open sourcemap of, for example, CategoryPage.style.scss, file in the browser _media.scss will be imported from the vendor.

**Comment by yeegor** [comment]
Once again, the theme should be able to launch on its own, in remote, without any dependencies other than within the theme itself. Currently it is the situation of using core build with remote backend, it can be launched from wherever. you want. It should remain this way. Please fix this.

This should help you:
image

**Review by yeegor — CHANGES_REQUESTED** [review]
See message above.

**Comment by ejnshtein** [comment]
Okay thanks, working on it.

**Comment by ejnshtein** [comment]
Either way with our example for `webpack.development.config.js` it compiles fine.
So what's the problem then?🤔

**Comment by ejnshtein** [comment]
OK, that should be it.

**Review by yeegor — CHANGES_REQUESTED** [review]
Please go through your changes and validate that nothing changed stylewise in neither desktop nor mobile version.

Cart page:
image

**Comment by ejnshtein** [comment]
Should be it.

**Review by yeegor — APPROVED** [review]
Seems OK.

**Pull request merged**